ironfish-cli/src/commands/wallet/import.ts

```diff
@@ -119,8 +120,13 @@ export class ImportCommand extends IronfishCommand {
   async importLedger(): Promise<string> {
     try {
       const ledger = new LedgerSingleSigner(this.logger)
-      await ledger.connect()
-      const account = await ledger.importAccount()
+
+      const account = await ui.ledger({
+        ledger,
+        message: 'Import Wallet',
+        action: () => ledger.importAccount(),
+      })
+
```

Comment:
nit - it looks like import needs some approval, should we add `approval: true`?

Reply:
Done, thanks for auditing which ones needed approval.